src/main/java/uk/gov/ons/ctp/response/sample/endpoint/SampleEndpoint.java

```diff
@@ -136,37 +136,6 @@ public SampleSummary ingest(final MultipartFile file, final String type) throws
     return newSummary;
   }
 
-  @RequestMapping(
-      value = "/{type}/fileupload",
-      method = RequestMethod.POST,
-      consumes = "multipart/form-data")
-  public ResponseEntity<SampleSummary> uploadSampleFile(
-      @PathVariable("type") final String type, @RequestParam("file") MultipartFile file)
-      throws CTPException {
-    log.debug("Entering Sample file upload", kv("sample_type", type));
-    if (!"B".equals(type.toUpperCase())) {
-      return ResponseEntity.badRequest().build();
-    }
-
-    try {
-      SampleSummary result = ingest(file, type);
```

Comment:
could you also remove this `ingest` fucntion? That function calls an ingest function in the SampleService class. that will need to be remove too and anything else within those functions that relate to file processing